content/content-types/src/webapp/WEB-INF/components.xml

```diff
@@ -4,7 +4,7 @@
     xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd">
 
 	<bean id="ContentTypeRegistyBean" class="org.sakaiproject.content.types.ContentTypeRegistryBean"
-	init-method="init">
+	init-method="init" depends-on="org.sakaiproject.localization.util.ContentTypeProperties">
```

Comment:
```suggestion
	init-method="init" depends-on="org.sakaiproject.localization.util.TypeProperties">
```

This should be TypeProperties even though its ContentTypeRegistryBean. This service uses some of these classes in content that depend on this class.